db/migrate/20210322044253_create_rulebooks.rb

```diff
@@ -0,0 +1,11 @@
+class CreateRulebooks < ActiveRecord::Migration[6.1]
+  def change
+    create_table :rulebooks do |t|
```

Comment:
sorry I added it later, but could you please also include an `img_url` column for the rulebook? /+ validations and tests)

Reply:
I only added a presence validation for the moment, not sure if we want to check for formats as well?

Reply:
let's check also formats just in case (ending in .jpg, .jpeg, .png, etc.)